nncf/common/pruning/weights_flops_calculator.py

```
@@ -106,35 +110,45 @@ def count_flops_and_weights_per_node(
             name = node.node_name
             if name in op_addresses_to_skip:
                 continue
-            num_in_channels = input_channels.get(name, node.layer_attributes.in_channels)
-            num_out_channels = output_channels.get(name, node.layer_attributes.out_channels)
-            kernel_size = kernel_sizes.get(name, node.layer_attributes.kernel_size)
+            layer_attr = node.layer_attributes
+            if not isinstance(layer_attr, ConvolutionLayerAttributes):
+                continue
```

Let's raise an internal error here

---

done